articles/communication-services/concepts/troubleshooting-info.md

```diff
@@ -353,6 +353,7 @@ The Azure Communication Services SMS SDK uses the following error codes to help
 | 4007 | The Destination/To number has opted out of receiving messages from you| Mark the Destination/To number as opted out so that no further message attempts are made to the number|
 | 4008 | You've exceeded the maximum number of messages allowed for your profile| Ensure you aren't exceeding the maximum number of messages allowed for your number or use queues to batch the messages |
 | 4009 | Message is rejected by Microsoft Entitlement System| Most often it happens if fraudulent activity is detected. Please contact support for more details |
+| 4010 | Message was blocked due to the toll free number not being verified | Review unverified sending limits and submit TFN for verification as soon as possible |
```

Comment:
Can you please hyperlink the toll-free verification FAQ where you say "review unverified sending limits".  

Also change "TFN" to "Toll-free verification" as soon as possible.